src/test/java/com/puppycrawl/tools/checkstyle/DefaultLoggerTest.java

```diff
@@ -166,4 +199,114 @@ public void testFinishLocalSetup() {
         assertNotNull(dl, "instance should not be null");
     }
 
+    /**
+     * Verifies that the language specified with the system property {@code user.language} exists.
+     */
+    @Test
+    public void testLanguageIsValid() {
+        final String language = DEFAULT_LOCALE.getLanguage();
+        if (!language.isEmpty()) {
```

Comment:
Why we need this condition, it can make test method without assert

Reply:
Please see https://github.com/checkstyle/checkstyle/issues/7170